File: day044/homework/code/codewars5.py
# https://www.codewars.com/kata/5583090cbe83f4fd8c000051/train/python
def digitize(n):
    res=[]
    n=str(n)
    n=n[::-1]
    for i in n:
        i=int(i)
        res.append(i)
    return res

File: day068/homework/code/codewars5.js
// https://www.codewars.com/kata/5b39e3772ae7545f650000fc/train/python

def remove_duplicate_words(s):
    seen = []
    for word in s.split():
        if word not in seen:
            seen.append(word)
    return ' '.join(seen)
